src/global/authorization/UserContext.tsx

```diff
@@ -79,10 +78,21 @@ export const UserProvider = ({ children }) => {
 		}) ;
 		navigate("/") ;
 	}
+	
+	const test = () => {
```

Comment:
@jonbondamon Look at these lines and checkout the simple test I made... Basically we just return data which is a special attribute containing whatever we return. In this case, I returned an object {'string':'Hello World'} and then accessed the specific "string" attribute of that object. If I had done console.log(data) it would have logged the entire object